sn_networking/src/event/swarm.rs

```diff
@@ -300,13 +304,15 @@ impl SwarmDriver {
                     }
                 }
             }
-
             SwarmEvent::NewListenAddr {
                 mut address,
                 listener_id,
             } => {
                 event_string = "new listen addr";
 
+                info!("Local node is listening {listener_id:?} on {address:?}");
+                println!("Local node is listening on {address:?}"); // TODO: make it print only once
```

Comment:
is this printout really necessary ?
could pollute cli terminal a lot ?

Reply:
We were already doing that, I just moved the line to the top. Maybe I'll remove it, I searched if we were using that stdout anywhere, but can't find any results.